### ISSUE TITLE
More info

### ISSUE THREAD

**Issue by noursicain** [dotnet/docs#9731]
The topic is Ok if you try to build an .exe. However i am trying to build a dll and i have this error. A link to the way of specifying dll or exe would have been nice.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: dcc1ba33-60dd-1e44-2498-eb76f0d964c1
* Version Independent ID: a1404161-c48b-564b-03ea-d4340fe28c53
* Content: [Compiler Error CS5001](https://docs.microsoft.com/en-us/dotnet/csharp/misc/cs5001)
* Content Source: [docs/csharp/misc/cs5001.md](https://github.com/dotnet/docs/blob/master/docs/csharp/misc/cs5001.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
@noursicain Can you post a smaller example somewhere? You should not get this error if you are building a DLL instead of an exe. (A DLL should not need an entry point).

**Comment by noursicain** [comment]
Hello,

Sorry it was so long ago that I have forgotten how I was able to fix it.

It happened when I tried to convert a exe project to a dll project in erasing the program.cs and in setting the output to dll. 

My point is the doc should have a link to information of things to check if the dll project act as an exe project.

If only  the output type control that my request is less useful. However if 2 ou 3 thing can tell the compiler that a project is an exe the list of things to check would be useful.

Thanks
